cdap-docs/developers-manual/source/building-blocks/mapreduce-programs.rst

```diff
@@ -20,12 +20,12 @@ Application specification::
     ...
     addMapReduce(new WordCountJob());
     
-or specify ``addWorkflow()`` in your Application and specify your MapReduce Job in the
+or specify ``addWorkflow()`` in your Application and specify your MapReduce in the
 :ref:`Workflow <workflows>` definition::
 
   public void configure() {
     ...
-    // Run a MapReduce Job on the acquired data using a Workflow
+    // Run a MapReduce on the acquired data using a Workflow
```

Comment:
Run a MapReduce on ? Or should it be run a mapreduce program? I am ok with former if that is used commonly.